wbooks/src/redux/books/reducers.ts

```diff
@@ -0,0 +1,24 @@
+import { BOOKS_MOCK } from '@constants/mockBooks';
+import { actionNames as bookActionNames } from '@redux/books/actions';
+import { AppState } from '@interfaces/AppInfo';
+import { IBookAction } from '@interfaces/BookInfo';
+
+const initialState: AppState = {
+  bookList: [],
+  idSelectedBook: null
+};
+
+const reducerBooks = (state = initialState, action: IBookAction): AppState => {
```

Comment:
Note: We use the reducer only for store variables, all the logic must be in the actions